okio/src/jvmMain/kotlin/okio/ByteString.kt

```diff
@@ -156,6 +157,13 @@ internal actual constructor(
     byteCount: Int
   ): Boolean = commonRangeEquals(offset, other, otherOffset, byteCount)
 
+  actual open fun copyInto(
```

Comment:
Need overloads?

Reply:
I think I’d prefer to omit them just because in Java you’ll be calling this with a bunch of int arguments and having lots of overloads with lots of int parameters feels somehow less ergonomic.